tests/test_model_validator.py

```diff
@@ -43,6 +43,32 @@ def val_model(cls, values: Any, info: ValidationInfo) -> dict[str, Any] | Model:
                 values.y += 1
             return values
 
+    m = Model(x=1, y=2)
+    assert m.model_dump() == {'x': 2, 'y': 3}
+    # model not changed because we don't revalidate m
+    assert Model.model_validate(m).model_dump() == {'x': 2, 'y': 3}
```

Comment:
Is this the change of behavior? Does this mean that we don't run before model validators if it's already a model?

Reply:
yes, will add. This is `test_model_validator_before` changing.

Reply:
fixed.